_data/simple-icons.json

```diff
@@ -10227,6 +10227,11 @@
             "hex": "FD5750",
             "source": "https://serverless.com/"
         },
+        {
+            "title": "Session",
+            "hex": "000000",
+            "source": "https://getsession.org/assets/downloads/Session-Brandmarks.zip"
```

Comment:
Please use only `https://getsession.org` for the source since we can found the direct links in the footer.